src/components/JSONOutput.js

```diff
@@ -18,15 +20,15 @@ export default function JSONOutput(props) {
   //Checks to insure the shouldRunSUSHI is true (aka button has been pressed) and there is text to display for the output
   if (props.shouldDisplaySUSHI && props.text) {
     return (
-      <Box className={classes.box} border={1}>
-        <h4>Your Output:</h4>
+      <Box className={classes.box} border={1} overflow="scroll">
+        <h4>Your Output: </h4>
         <p>{props.text}</p>
```

Comment:
@cmoesel has taught me the wonders of `<pre>`. If you replace this `<p>` tag with `<pre>` the JSON will be spaced out with the `\t` you gave it in `JSON.stringify()`.